### ISSUE TITLE
dF computation

### ISSUE THREAD

**Issue by shinkira** [cortex-lab/Suite2P#79]
To compute dF, extractSignals.m used to call my_ica.m to estimate coefNeu (coefficient for neuropil subtraction). On the current master branch, my_ica is no longer called (in getActivityStats.m) and a default value of coefNeu (=0.7) is used for all ROIs. I am wondering why such a change was made and why coefNeu is no longer estimated for individual ROIs.

**Comment by marius10p** [comment]
The neuropil coefficients are now computed together with the spike deconvolution, because we got better performance than with ICA. This step is fast (even fast than the ICA). A default value is set to 0.7 before spike deconvolution is ran. Any feedback on the new method is appreciated.

**Comment by shinkira** [comment]
Thank you for the clarification. Then it would be better to call getActivityStats.m again after spike deconvolution to update stat for each ROI?

**Comment by marius10p** [comment]
The deconvolution adds this new neuropil coefficient directly into the saved F files.

**Comment by shinkira** [comment]
Yes, but other stats (skew, maxMinusMed, top5pcMinusMed) are not updated with the estimated neuropil coefficient. Perhaps these stats are not as important as neuropilCoefficient.

**Comment by marius10p** [comment]
True. Should not matter much though: those are only used in the auto classification of the ROIs and for display purposes in the GUI.
